instruction-error/src/lib.rs

```diff
@@ -1,13 +1,11 @@
 #![no_std]
 #![cfg_attr(feature = "frozen-abi", feature(min_specialization))]
-use core::fmt;
 #[cfg(feature = "num-traits")]
 use num_traits::ToPrimitive;
 #[cfg(feature = "frozen-abi")]
 use solana_frozen_abi_macro::{AbiEnumVisitor, AbiExample};
-#[cfg(feature = "std")]
+#[cfg(feature = "frozen-abi")]
 extern crate std;
```

Comment:
will this be confusing since std is included even without a `std` feature?

i guess my expectation for a `no-std` crate is that it doesn't use `std` unless you pass `std` feature.

Reply:
frozen-abi doesn't count imo, if someone pulls that in they have other problems